.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
+icgc_download_client.egg-info
```

Comment:
That, like build/ and dist/ is a byproduct of using python setup.py install.  The directory contains metadata for the package.  It still has the old name because my attempts to change the base directory name for my project have been met with error messages.  I added all three to gitignore because it was annoying to delete them all and reinstall every time I commited